src/scripts/core/visualization-container.js

```diff
@@ -74,6 +74,12 @@
             return this.ctx;
         },
 
+        setVisibility: function (visible) {
+            var node = this.layer.node();
+
+            visible ? $(node).show() : $(node).hide();
```

Comment:
I think we may have different definitions of what contour dependencies are. Regardless, for me, if jquery is available on the global scope, then I don't see the harm in using it.

I guess what I'm trying to say is, I'm not sure what you're trying to say.